Fits/T1 EWAR Cruisers/[NVY-30] Celestis.md

```diff
@@ -0,0 +1,46 @@
+# Celestis
+
+Entry EWAR Cruiser
+
+## Description
+
+Utilizes Remote Sensor Dampeners (Damps) to disrupt hostile DPS or Logistics. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary.
+
+Targeting Range - Use on long range platforms like enemy ECM or ships with long engagement ranges.
+Scan Resolution - Use on Logistics ships that are close to the enemy fleet or against any ship that is too close for targeting range to make a difference.
+
+## Fit
+
+```
+[Celestis, [NVY-30] Celestis]
```

Comment:
Similar to Arbitrator, this ship can actually survive if you fit it like a brick.